### ISSUE TITLE
Add term bulging fontanelle

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10403]
**Preferred term label:**
Bulging fontanelle

**Synonyms**
Tense fontanelle

**Definition (free text, please give PubMed ID)**
An outward curving of a fontanelle, which occurs when fluid builds up in the brain or the brain swells, causing increased pressure inside the skull. PMID: 12825844

**Parent term (use hpo.jax.org/app)**
Abnormality of fontanelles HP:0011328

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
For example: Meningitis (ICD10 G03.9), Encephalitis (ICD10 G04.90), Hydrocephalus (ICD10 G91.9)
See PMID: 12825844 for a differential diagnosis 

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw 
Adding new term: 
- ID: HP:6000647
- label: Bulging fontanelle
- parents: [Abnormality of fontanelles (HP:0011328)](https://hpo.jax.org/app/browse/term/HP:0011328)
- pmid: PMID:12825844
- def: An outward curving of a fontanelle, which occurs when fluid builds up in the brain or the brain swells, causing increased pressure inside the skull.
- comment: 
- orcid: orcid:0009-0007-1020-5412
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10403
- synonyms: Tense fontanelle(NONE)
